ReactWindows/ReactNative/Bridge/Queue/LayoutActionQueue.cs

```diff
@@ -8,12 +10,37 @@ namespace ReactNative.Bridge.Queue
     class LayoutActionQueue : DispatcherActionQueue
     {
 #if CREATE_LAYOUT_THREAD
-        private static readonly CoreApplicationView s_layoutApplicationView = CoreApplication.CreateNewView();
+        private const int E_ILLEGAL_METHOD_CALL = -2147483634; // 0x80000000E
 
+        private static readonly Lazy<CoreApplicationView> s_layoutApplicationView = new Lazy<CoreApplicationView>(() =>
```

Comment:
>private [](start = 8, length = 7)

Is this change worth making in a separate PR if its not specific to multi-window scenarios? It seems like it could be useful in single-window custom activation scenarios.

Reply:
I don't think it's possible since "multi-window" also covers the "one window that is not main window". In other words prior code would explode in other places if activated this way.

---
In reply to: [166566707](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166566707) [](ancestors = 166566707)